### ISSUE TITLE
Facing some webdriver exceptions

### ISSUE THREAD

**Issue by sathwik04** [shaikhsajid1111/facebook_page_scraper#5]
inside facebook_page_scraper i have runned setup.py as per instrcutions.
 when iam trying to run the  same example iam getting an error:
[WDM] - Driver [/root/.wdm/drivers/geckodriver/linux64/v0.29.0/geckodriver] found in cache
Traceback (most recent call last):
  File "face.py", line 8, in <module>
    json_data = facebook_ai.scrap_to_json()..

  raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: Process unexpectedly closed with status 1

can anyone figure out this error and provide me running example .

**Comment by shaikhsajid1111** [comment]
Which browser is installed on your system?, Firefox or Chrome?. If you are using same code as mentioned in README then it means you have firefox installed on your default path?. 

Similar question on stackoverflow

**Comment by sathwik04** [comment]
Yes firefox is installed in my system and iam running example as mentioned in Readme file 
Inside facebook-scrapper folder Iam trying to run script. 
Can you suggest me how to solve this issue. 
It is little bit important for me.

**Comment by shaikhsajid1111** [comment]
I wonder if its a driver compatibility issue. [See](https://stackoverflow.com/questions/46809135/webdriver-exceptionprocess-unexpectedly-closed-with-status-1)
